datahub/company/tasks.py

```diff
@@ -26,11 +26,17 @@ def _automatic_company_archive(task, limit, simulate):
         latest_interaction_date=Subquery(
             latest_interaction.values('date')[:1],
         ),
+        active_investment_projects=FilteredRelation(
+            'investor_investment_projects',
+            condition=Q(investor_investment_projects__status='ongoing'),
```

Comment:
Would be good to use the named constant for 'ongoing' i.e.:
```suggestion
            condition=Q(investor_investment_projects__status=InvestmentProject.STATUSES.ongoing),
```